docs/content/download/default.yml

```
@@ -24,7 +24,7 @@ body:
          `sudo apt-get install jq`.
 
        * jq 1.5 is in the official
```

It's actually at 1.6.4 now in Fedora 32

---

We haven't released any revisions to 1.6- that `-4` is referring to the version of the RPM itself. That said, this _should_ say 1.6 instead of 1.5. Mind updating it?

---

@wtlangford , since this seems to be a dead PR.  I've confirmed here https://src.fedoraproject.org/rpms/jq/blob/master/f/sources that Fedora is using jq v1.6

```suggestion
       * jq 1.6 is in the official
```
Can you accept the GH suggestion and we get this closed, or would you prefer I make a new PR?

---

> We haven't released any revisions to 1.6- that `-4` is referring to the version of the RPM itself. That said, this _should_ say 1.6 instead of 1.5. Mind updating it?

Apologies, I thought I had already done the update and you had to accept the PR?

---

> Apologies, I thought I had already done the update and you had to accept the PR?

@calpeconsulting The change in the PR still refers to jq 1.5.  It needs to be jq 1.6.